spec/minesweeper_spec.rb

```diff
@@ -0,0 +1,328 @@
+require('minesweeper')
+
+describe Minesweeper do
+	describe "#initialize" do
+		it 'creates a new board object' do
+			expect(Board).to receive(:new).with(10,9)
+			Minesweeper.new
+		end
+
+		# QUESTION: Why doesn't this work when I remove the let and before statements above?
+		it 'creates a new player object' do
+			expect(Player).to receive(:new)
+			Minesweeper.new
+		end
+	end
+end
+
+describe Board do
+	let(:new_board){Board.new(10,9)}
+	let(:s){new_board.get_square([1,1])}
+
+  describe "#initialize" do
```

Comment:
Could add a couple more checks here... even confirming the passed size is correct is good just in case